### PR TITLE
Getting rid of test script + Adding TRAP

### PR DESCRIPTION
**Pull request by jhadvig**
Making hack/ dir DRY-er
@soltysh updating this as promised :)

### CODE REVIEW

File: hack/build.sh [hunk 1]
```diff
@@ -1,35 +1,54 @@
...
+
+  docker build -t ${TAG} . && trap - ERR
+}
 
 if [ -z ${VERSION} ]; then
   # Build all versions
   dirs=${VERSIONS}
 else
-  # Build only specified version on NodeJS
+  # Build only specified version on Ruby
```
Comment (soltysh) [inline hack/build.sh]: I guess, we're still in NodeJS repo :wink:

File: hack/build.sh [hunk 2]
```diff
@@ -1,35 +1,54 @@
...
   dirs=${VERSION}
 fi
 
-for dir in ${dirs}; do
-  IMAGE_NAME=openshift/nodejs-${dir//./}-${OS}
-
+for dir in ${dirs[@]}; do
+  IMAGE_NAME=openshift/ruby-${dir//./}-${OS}
```
Comment (soltysh) [inline hack/build.sh]: Same here

**Comment by soltysh** [comment]
Other than nasty ruby, LGTM.

**Comment by jhadvig** [comment]
@soltysh updated :) thanks

**Comment by bparees** [comment]
lgtm.

**Pull request merged**